api/lib/domain/models/Assessment.js

```diff
@@ -72,6 +80,7 @@ class Assessment {
     this.userId = userId;
     this.competenceId = competenceId;
     this.campaignParticipationId = campaignParticipationId;
+    this.method = method;
```

Comment:
proposition : 
```suggestion
    this.method = method || computeMethodFromType(this.type);
```
Pour s'assurer d'avoir toujours la méthode bien remplit si on envoie rien.